src/Utils/Image.cs

```diff
@@ -301,29 +289,15 @@ public Image ConvertTo(string format, string metadata = null, int dpi = 0)
                 ext = "png";
                 break;
             case "JPG":
-                img.SaveAsJpeg(ms, new JpegEncoder() { Quality = 100 });
-                break;
-            case "JPG90":
-                img.SaveAsJpeg(ms, new JpegEncoder() { Quality = 90 });
```

Comment:
same here as above - anyone who had JPG90 selected needs to not have their swarm suddenly break because of this change.